enterprise/server/cmd/ci_runner/main.go

```diff
@@ -694,6 +697,35 @@ func readConfig() (*config.BuildBuddyConfig, error) {
 	return c, nil
 }
 
+func getDefaultConfig() *config.BuildBuddyConfig {
+	// By default, test all targets when any branch is pushed.
+	// TODO: Consider running a bazel query to find only the targets that are
+	// affected by the changed files.
+	triggerBranches := []string{*triggerBranch}
+	return &config.BuildBuddyConfig{
+		Actions: []*config.Action{
+			{
+				Name: "Test all targets",
+				Triggers: &config.Triggers{
+					Push: &config.PushTrigger{Branches: triggerBranches},
+				},
+				BazelCommands: []string{
+					// TOOD: Consider enabling remote_cache and remote_executor along with
+					// recommended RBE flags.
+					fmt.Sprintf(
+						"test //... "+
+							"--build_metadata=ROLE=CI "+
+							"--bes_backend=%s --bes_results_url=%s "+
+							"--remote_header=x-buildbuddy-api-key=%s",
```

Comment:
It's been a while since I looked at the CI code but I thought we were injecting API key at a lower level than this -- does it need to be in this config?

Reply:
Ah yeah, we can set `BUILDBUDDY_WORKFLOW_ID` like we do for the "outer" invocation and that should work. (Done).

Reply:
NVM, `BUILDBUDDY_WORKFLOW_ID` is only used for the GitHub access token. Let me look through this some more...

Reply:
For the "outer" invocation we are setting the API key via the BEP client ctx, but for the inner invocation I think we have to do it via bazel flags.

Reply:
ok, wfm.